static/js/containers/FinancialAidCalculator.js

```diff
@@ -109,6 +124,16 @@ const validationMessage = (key, errors) => {
   </div>;
 };
 
+const apiError = ({ message, code }: FetchError) => (
+  <div className="api-error">
```

Comment:
Hmm, that seems like a problem as well. I believe that HTML form elements are supposed to always be enclosed in a `<form>` tag, otherwise the HTML is invalid. But that's another problem for another day. Sounds like this is way out of scope for this pull request, so we can just leave it as-is for now.